app/models/grda_warehouse/service_history.rb

```diff
@@ -79,41 +79,47 @@ def readonly?
   end
```

Comment:
Let's delete this model and see what breaks.

Reply:
Moved to separate branch